app/components/Modal/MarketPickerHelpers.js

```diff
@@ -0,0 +1,145 @@
+import {hasGatewayPrefix} from "common/gatewayUtils";
+import {ChainStore} from "bitsharesjs";
+
+function lookupAssets(value, gatewayAssets = false, getAssetList, setState) {
+    if (!value && value !== "") return;
+
+    let quote = value.toUpperCase();
+
+    if (quote.startsWith("BIT") && quote.length >= 6) {
```

Comment:
no asset can start with BIT. That prefix is frontend only

Reply:
I've copied it from here as is, perhaps you need to fix that too in case it's wrong: https://github.com/bitshares/bitshares-ui/blob/develop/app/components/Exchange/MarketPickerHelpers.js

Reply:
If user search for bitBTC there isn't any, so make it BTC, one doesn't have to know about it.
It is used for searching.